renku_notebooks/api/classes/cloud_storage/existing.py

```diff
@@ -1,59 +1,37 @@
 from dataclasses import dataclass
 from typing import Any, Dict, List
 
-from ....config import config
-
 
 @dataclass
 class ExistingCloudStorage:
-    bucket: str
-    endpoint: str
+    remote: str
+    type: str
```

Comment:
Not the end of the world, but it feels like it may be a good idea to make this an Enum?

Or will this then have to have every possible value that Rclone supports?

Reply:
Also a general question here:
Is this now capturing old cloud storage - backed by datashim as well as new stuff backed by rclone? If not how do we support datashim on sessions that already exist?

Reply:
since this is only for showing cloudstorage for sessions and the UI doesn't show cloud storage, it's irrelevant. I remember we discussed this.